modules/schema/pages/procedure-lot-part-information.adoc

```diff
@@ -1883,13 +1882,42 @@ within that Award Criteria no Award Criterion with Award Criterion Number Weight
 </cac:AwardingCriterion>
 ----
 
-When Award Criteria calculation is too complex to be described with the proposed structure, 
+When Award Criteria calculation is too complex to be described with the 
+proposed structure including Award Criterion Number (BT-541), 
 the complicated expression, or a reference to the procurement document section where this 
 information may be found, should be specified in Award Criteria Complicated (BT-543). 
 Information is encoded as text with the element "_cbc:CalculationExpression_". For a given 
 Award Criteria (i.e. "_cac:AwardingCriterion_" element), the children "_cbc:Description_" 
 (Award Criteria Order Justification, BT-733) and "_cbc:CalculationExpression_" (Award 
-Criteria Complicated, BT-543) shall not coexist.
+Criteria Complicated, BT-543) shall not coexist. Similarly, for a given Award 
+Criteria (i.e. "_cac:AwardingCriterion_" element), the child element 
+"_cbc:CalculationExpression_" (Award Criteria Complicated, BT-543) and the 
+descendant element "_efbc:ParameterNumeric_" (Award Criterion Number, BT-541) 
```

Comment:
"shall not coexist with BT-541, 5421, 5422 or 5423" or something equivalent. In fact in the rules 541 since 5421, 5422, 5423 also depends from 541. It may however be clearer to state it at once. Up-to you